src/Spec-BackendTests/AbstractTreeTableAdapterTest.class.st

```diff
@@ -72,12 +72,11 @@ AbstractTreeTableAdapterTest >> testSelectingAPathUpdatesSelectionInPresenter [
 ]
 
 { #category : #running }
-AbstractTreeTableAdapterTest >> testSelectingOutOfRangeResetsSelection [
-
+AbstractTreeTableAdapterTest >> testSelectingOutOfRangeKeepOldSelection [
 	self presenter selection selectedPath: #(2 3).
 	self presenter selection selectedPath: #(7 3).
 	
-	self assert: self adapter selectedItem equals: nil
+	self assert: self adapter selectedItem equals: 20
```

Comment:
I changed the expected result because this is the behavior of the list selection.